pkg/api/validation/schema.go

```diff
@@ -311,6 +311,9 @@ func (s *SwaggerSchema) validateField(value interface{}, fieldName, fieldType st
 		return s.ValidateObject(value, fieldName, fieldType)
 	}
 	allErrs := []error{}
+	if reflect.TypeOf(value) == nil {
```

Comment:
How about `reflect.ValueOf(value).IsNil()`? Also I would expect this check to be at the top of this function.